pkg/sethandler/controller.go

```diff
@@ -99,6 +99,12 @@ func shouldPodBeHandled(pod v1.Pod) bool {
 }
 
 func (setHandler *SetHandler) adjustContainerSets(pod v1.Pod) {
+	for _, containerStatus := range pod.Status.ContainerStatuses {
+		if !containerStatus.Ready {
+			log.Println("INFO: container " + containerStatus.Name + " not READY!")
```

Comment:
this log entry might be too overzealous considering the traffic of the Pod API, please delete